### DISCUSSION TITLE
Using avg value instead of last value in recurring transactions.

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#110] [Ideas]
What title said.

**Comment by ananthakumaran** [comment]
Let me think, an argument could be made for both last and average. Probably some way to make it configurable would be better.

**Reply by avinashkanaujiya** [reply]
Yes, that should work.

**Reply by avinashkanaujiya** [reply]
Also, the avg should only include recent transactions like 3 or 6 months, to make transaction reflect recent increased cost.
